Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorEffects/SegmentEditorDrawEffect.py

```diff
@@ -32,13 +34,13 @@ def icon(self):
         return qt.QIcon()
 
     def helpText(self):
-        return """<html>Draw segment outline in slice viewers<br>.
-<p><ul style="margin: 0">
-<li><b>Left-click:</b> add point.</li>
-<li><b>Left-button drag-and-drop:</b> add multiple points.</li>
-<li><b>x:</b> delete last point.</li>
-<li><b>Double-left-click</b> or <b>right-click</b> or <b>a</b> or <b>enter</b>: apply outline.</li>
-</ul><p></html>"""
+        return _("""<html>Draw segment outline in slice viewers<br>.
+        <p><ul style="margin: 0">
```

Comment:
This means the indentation will be part of the string.

![image](https://github.com/Slicer/Slicer/assets/219043/2a97c98a-8a62-4885-b4de-6b7e70aa67fd)

To help with this, we have been using `textwrap.dedent`


![image](https://github.com/Slicer/Slicer/assets/219043/92d7fe8a-2bfe-4592-9e04-d6c1419f5e36)

I suggest to keep the text without any indentation